broker/lib/bosh/BoshDirectorClient.js

```diff
@@ -621,6 +621,28 @@ class BoshDirectorClient extends HttpClient {
 
   /*  Task operations */
 
+  getTasksForDashboard(options) {
```

Comment:
as discussed , pls see if we can re-use existing `getTask(option)` but refactoring it to `getTask(option,fetchDirectorForDeployment)`